torch/csrc/distributed/rpc/script_call.cpp

```diff
@@ -22,29 +22,36 @@ const std::vector<at::IValue>& ScriptCall::stack() const {
 
 Message ScriptCall::toMessage() {
 
-  std::vector<IValue> ivalues;
+  // std::vector<IValue> ivalues;
+  // c10::impl::GenericList list(c10::impl::deprecatedUntypedList());
+  c10::List<IValue> list = c10::impl::GenericList(c10::impl::deprecatedUntypedList());
```

Comment:
Why did this change from a tuple to a list? I list is not appropriate here because the types are not the same, what would the type of the list be?